drafts/WEP0001.rst

```diff
@@ -0,0 +1,273 @@
+  and therefore the latter should be scored higher.
+* Where is this person from? If they would take a large budget, but live in an
+  area that has many other conferences closer by, they may not be a great choice.
+  So all other things being equal, we would rather spend € 1000 on an attendee
+  from Kenia than from Portland, Oregon, US.
+* Are they making a basic effort? For example, if someone doesn't even have an
+  estimate of the price of their flight without further explanation, does not
+  appear very motivated.
+* Does this conference seem like an event that would benefit them? And do
+  they have some understanding of what the conference is about?
+
+After all scores are completed, the grant review team will discuss the top
+applications, prioritised by underrepresented/marginalised groups first,
+score second, and grant them unless discussion leads to another conclusion,
+until funds run out. The review team is not required to always follow the
+score based order. This also depends on the remaining resources.
+
+Speakers will bypass the review process and are immediately accepted, except
+in exceptional circumstances.
+
+Notifying accepted applicants
+-----------------------------
+
+* All accepted applicants will be notified by the grant review team.
+* Applicants need to confirm they are still intending to participate.
+* If they withdraw their application, the grant review team may be able
+  provide a few extra applicants with grants, prioritised by score.
+* Applicants that are very low on the list, and basically certain not to
+  receive a grant, will be notified by the grant review team as soon as
+  the selection is complete.
+* When all accepted applicants are confirmed, the remaining not accepted
+  applicants will be notified too.
+
+Handling grants and funds
+-------------------------
+
+* Accepted applicants are encouraged to complete their bookings as soon
+  as possible.
+* Some may need to request a visa. In that case, the grant review team
+  should stress that they should not book non-refundable travel or
+  accommodation until their visa is confirmed. Upon request, an invitation
+  letter will be provided, which should mention that costs are covered
+  by the conference, as this can help with the visa application.
+* Applicants need to provide booking confirmations for their costs.
+  Upon receipt of the confirmations (which need to match their legal name),
+  their costs will be refunded as soon as possible (within a few weeks).
+  There may be costs involved in these transfers, which should be covered
+  by the conference.
+* In circumstances where it is not realistically possible, or prohibitively
+  expensive, to transfer the funds, the refund can happen at the conference.
+  This should be an exceptional case though, as not everyone can afford to
+  miss these funds so long.
+* In very exceptional circumstances, transfers may be made before booking
```

Comment:
```suggestion
* In very exceptional circumstances, transfers may be made before booking
```

Just out of curiosity, is this more feasible than WTD or an external provider paid by WTD booking the tickets for them?

Reply:
We can add that as an option. Historically, I've been reluctant and nervous to book flights for other people, but it should be a possibility.

Reply:
This has been added, thanks :)